rsyslog.service.in

```diff
@@ -1,8 +1,10 @@
 [Unit]
 Description=System Logging Service
-Requires=syslog.socket
+;Requires=syslog.socket
+Wants=network.target network-online.target
```

Comment:
Aside from not really agreeing with the changes, `Wants=network.target` doesn't really make sense.
network.target is a passive unit and one should not actively pull that in via Wants. 
See man systemd.special for more information.

Reply:
Or https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/